### PR TITLE
CLIP-1747: Update Postgres and Elasticsearch helm charts

### PR DESCRIPTION
**Pull request by bianchi2**
Bitnami have removed postgresql 10.16.2 and elasticsearch 17.9.29 from their indexes.

This PR bumps bitnami/postgresql chart version, and replaces bitnami/elasticsearch chart with an official elastic chart.

Had to completely replace postgres and es values to match new helm charts.

As a result, postgres 14 is deployed, while Elasticsearch version remained unchanged.

## Checklist
- [ ] I have added unit tests
- [ ] I have applied the change to all applicable products
- [ ] I have added the change description to the `changelog.md` and `Chart.yaml` files
- [ ] (Atlassian only) I have run the E2E test (if applicable)
- [ ] (Atlassian only) Internal Bamboo CI is passing

### CODE REVIEW

File: src/test/infrastructure/elasticsearch/elasticsearch-values.yaml [hunk 1]
```diff
@@ -1,27 +1,15 @@
----
+persistence:
+  enabled: false
 
-global:
-  coordinating:
-    name: coo
+secret:
```
Comment (bianchi2) [inline src/test/infrastructure/elasticsearch/elasticsearch-values.yaml:4]: Auth is enabled by default. If we want to we may keep it enabled and pass secret name and keys to bitbucket.

File: src/test/infrastructure/elasticsearch/elasticsearch-values.yaml [hunk 2]
```diff
@@ -1,27 +1,15 @@
...
+  enabled: false
 
-master:
-  replicas: 1
-  podAnnotations:
-    "atlassian.com/business_unit": "Engineering: Enterprise DC"
-  persistence:
-    enabled: false
+createCert: false
```
Comment (bianchi2) [inline src/test/infrastructure/elasticsearch/elasticsearch-values.yaml:7]: Disable ssl, or else Bitbucket will need the cert it its keystore, which is an overkill for CI.

File: src/test/scripts/helm_install.sh [hunk 3]
```diff
@@ -126,9 +129,9 @@ bootstrap_database() {
      --values $PSQL_CHART_VALUES \
      --set fullnameOverride="$POSTGRES_RELEASE_NAME" \
      --set image.tag="$POSTGRES_APP_VERSION" \
-     --set postgresqlDatabase="$DB_NAME" \
-     --set postgresqlUsername="$PRODUCT_NAME" \
-     --set postgresqlPassword="$PRODUCT_NAME" \
+     --set auth.database="$DB_NAME" \
```
Comment (bianchi2) [inline src/test/scripts/helm_install.sh:132]: See: https://github.com/bitnami/charts/tree/main/bitnami/postgresql

**Review by nghazali — APPROVED** [review]

**Review by nanux — APPROVED** [review]

**Comment by nanux** [comment]
This is green as well - https://server-syd-bamboo.internal.atlassian.com/browse/DCD-K8SHELMTEST144-8

**Pull request merged**